imports/ui/pages/profile.jsx

```diff
@@ -13,6 +13,8 @@ import { ChangePasswordModal } from '../modals/ChangePasswordModal'
 import { Images } from '../../api/images'
 import { DragAndDropArea } from '../DragAndDropArea.jsx'
 
+import { Settings } from '../../api/settings'
```

Comment:
probably don't need to import settings... use a new method to get the image related parameters